src/Lecture1.hs

```diff
@@ -108,7 +121,12 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+
+toInt :: [String] -> [Int]
+toInt = map read
+
+strSum :: [Char] -> Int
+strSum str = sum (toInt (words str))
```

Comment:
As you specify the top-level type signature for this function, the compiler would be able to figure out what type `read` function has in this case due to type inference, so you can inline the definition of `toInt` directly here:

```suggestion
strSum str = sum (map read (words str))
```